src/relentless/simulate/hoomd.py

```diff
@@ -213,6 +213,33 @@ def _convert_to_gsd_file(self, sim):
             gsd_filename = mpi.world.bcast(gsd_filename)
         else:
             gsd_filename = self.filename
+            if mpi.world.rank_is_root:
```

Comment:
I was slightly confused here because this variable gets declared but then overridden on everything that's not the root rank. This made more sense to me:
```suggestion
            if mpi.world.rank_is_root:
                gsd_filename = self.filename
```